### PR TITLE
Add Kotlin to API documentation 2.0

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

This PR updates: 
1. API documentation with Kotlin syntax and examples
2. Some grammar/styling

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/api-reference.md [hunk 1]
```diff
@@ -82,25 +114,40 @@ public static void registerExtension()
 
 ##### Example
 ```java
-import com.adobe.marketing.mobile.Edge
+import com.adobe.marketing.mobile.Edge;
```
Comment (addb) [inline Documentation/api-reference.md:117]: Let's remove the imports from the sample. We have that in getting started.

File: Documentation/api-reference.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Adobe Experience Platform Edge Network Extension - Android
+# Adobe Experience Platform Edge Network Extension Android API Usage
```
Comment (emdobrin) [inline Documentation/api-reference.md:1]: or API reference?

File: Documentation/api-reference.md [hunk 3]
```diff
@@ -35,35 +35,66 @@ public static String extensionVersion()
...
+```kotlin
+val extensionVersion = EdgeBridge.extensionVersion()
+```
+
 ------
 
 ### getLocationHint
 
-Gets the Edge Network location hint used in requests to the Adobe Experience Platform Edge Network. The Edge Network location hint may be used when building the URL for Adobe Experience Platform Edge Network requests to hint at the server cluster to use.
+Gets the Edge Network location hint used in requests to the Edge Network. The location hint may be used when building the URL for Edge Network requests to hint at the server cluster to use.
```
Comment (emdobrin) [inline Documentation/api-reference.md:54]: can we keep `The Edge Network location hint` as it provides more context on which hint in particular will be used

File: Documentation/api-reference.md [hunk 4]
```diff
@@ -109,64 +156,105 @@ Sends an Experience event to the Adobe Experience Platform Edge Network
...
+val experienceEvent = ExperienceEvent.Builder()
+  .setXdmSchema(xdmData)
+  .build()
+```
+```kotlin
+// Example 1 - send the experience event without handling the Edge Network response
+Edge.sendEvent(experienceEvent, null)
+```
+```java
+// Example 2 - send the experience event and handle the Edge Network response onComplete
```
Comment (addb) [inline Documentation/api-reference.md:209]: 1. ```kotlin

2. nit: send the experience event and handle the Edge Network response `callback` onComplete

File: Documentation/api-reference.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Adobe Experience Platform Edge Network Extension - Android
+# Adobe Experience Platform Edge Network Extension Android API Usage
 
 ## Prerequisites
```
Comment (emdobrin) [inline Documentation/api-reference.md:4]: nit: below : Refer to the [Getting started guide](getting-started.md).

**Review by emdobrin — COMMENTED** [review]
Looks great, few small comments below

File: Documentation/api-reference.md (hunk 1 above)
Comment (kevinlind) [reply Documentation/api-reference.md:117]: The documentation for the other extensions do include the import statement just for the `registerExtension` API, so I think we can leave it here as well.

**Review by emdobrin — APPROVED** [review]
let's :shipit:

**Pull request merged**